src/homepageExperience/containers/ArduinoWizard.tsx

```diff
@@ -0,0 +1,219 @@
+// Libraries
+import React, {PureComponent} from 'react'
+import classnames from 'classnames'
+
+// Components
+import {
+  Button,
+  ComponentColor,
+  ComponentSize,
+  ComponentStatus,
+  Page,
+  SubwayNav,
+} from '@influxdata/clockface'
+import {ArduinoIcon} from 'src/homepageExperience/components/HomepageIcons'
+
+// Steps
+import {ExecuteAggregateQuery} from 'src/homepageExperience/components/steps/arduino/ExecuteAggregateQuery'
+import {ExecuteQuery} from 'src/homepageExperience/components/steps/arduino/ExecuteQuery'
+import {Finish} from 'src/homepageExperience/components/steps/Finish'
+import {InitializeClient} from 'src/homepageExperience/components/steps/arduino/InitializeClient'
+import {InstallDependencies} from 'src/homepageExperience/components/steps/arduino/InstallDependencies'
+import {Overview} from 'src/homepageExperience/components/steps/Overview'
+import {WriteData} from 'src/homepageExperience/components/steps/arduino/WriteData'
+import WriteDataDetailsContextProvider from 'src/writeData/components/WriteDataDetailsContext'
+
+// Utils
+import {event} from 'src/cloud/utils/reporting'
+import {HOMEPAGE_NAVIGATION_STEPS_SHORT} from 'src/homepageExperience/utils'
+import RateLimitAlert from 'src/cloud/components/RateLimitAlert'
+
+interface State {
+  currentStep: number
+  selectedBucket: string
+  finishStepCompleted: boolean
+  tokenValue: string
+  finalFeedback: number
+}
+
+export class ArduinoWizard extends PureComponent<{}, State> {
+  state = {
+    currentStep: 1,
+    selectedBucket: 'sample-bucket',
+    finishStepCompleted: false,
+    tokenValue: null,
+    finalFeedback: null,
+  }
+
+  private handleSelectBucket = (bucketName: string) => {
+    this.setState({selectedBucket: bucketName})
+  }
+
+  private handleMarkStepAsCompleted = () => {
+    this.setState({finishStepCompleted: true})
+  }
+
+  private setTokenValue = (tokenValue: string) => {
+    this.setState({tokenValue: tokenValue})
+  }
+
+  private setFinalFeedback = (feedbackValue: number) => {
+    this.setState({finalFeedback: feedbackValue})
+  }
+
+  handleNextClick = () => {
+    this.setState(
+      {
+        currentStep: Math.min(
+          this.state.currentStep + 1,
+          HOMEPAGE_NAVIGATION_STEPS_SHORT.length
+        ),
+      },
+      () => {
+        event(
+          'firstMile.arduinoWizard.next.clicked',
+          {},
+          {
+            clickedButtonAtStep: this.state.currentStep - 1,
```

Comment:
this doesn't need to be addressed in this pr, but we need to update the data we send here to be the name of the step instead of its index.   ```clickedButtonAtStep: normalizeEventName(HOMEPAGE_NAVIGATION_STEPS_SHORT[this.state.currentStep - 1].name)```